### DISCUSSION TITLE
用機器學習拯救海藻

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#133] [General]
Charlene Xia 開發了低成本的系統來監測海藻養殖場的微生物群系，並在疾病傳播之前識別疾病。
![MIT-Seaweed-Farming-01-PRESS_0](https://user-images.githubusercontent.com/91121500/139009276-344cc1ae-b0ef-412b-9ca8-3b1efaa9234c.jpg)

該團隊將舊有的技術與最新的計算技術相結合， 並使用潛水式數位全息顯微鏡（submersible digital holographic microscope）拍攝二維的圖像，接著運用神經網絡的機器學習系統將 2D 圖像轉換為 3D 表示，顯示出微生物群系（microbiome），並在疾病傳播之前提早識別並防範疾病的發生。

資料來源：https://news.mit.edu/2021/saving-seaweed-machine-learning-1022

**Comment by EricAgumonJou** [comment]
Charlene Xia 開發出的這些系統與設備成本低於 100 美元，（其他家公司開發的系統與設備成本約為4000多美元）
該低成本的價格，有助於降低那些有興趣揭開海洋許多奧秘的人的成本上的困難，該通信設備可從機器學習算法中取得有關海洋環境的數據。

**Comment by EricAgumonJou** [comment]
該團隊開發的軟體是可以在Raspberry Pi 中運行。
其中Raspberry Pi 可連接全息顯微鏡（holographic microscope）一起使用。

**Comment by HSU-YU-JEN** [comment]
原來機器學習也可以運用在監測海洋生態

**Comment by enoch-lightning** [comment]
有實際案例，便能對人工智慧的應用有更深入的了解。我有點疑惑的是，神經網路(neural network，NN)是深度學習(deep learning)才有，還是機器學習(machine learning)呢？

**Reply by EricAgumonJou** [reply]
實際例子請參閱資料來源～
兩者都有，深度學習是機器學習的其中一個子領域，關係圖如下圖，兩者均以人工神經網為架構進行表徵學習。
![ai_](https://user-images.githubusercontent.com/91121500/139032637-9fd7d14f-284b-441d-b7fc-0eba0fe32a7d.png)

**Reply by enoch-lightning** [reply]
我的意思是說，你有給實際例子，讓人感覺既新鮮又具體。:smile:

**Reply by EricAgumonJou** [reply]
拍謝😅，誤會了你的意思😂

**Reply by enoch-lightning** [reply]
@EricAgumonJou 想請問你如何在「拍謝」後面插入表情符號，以及在最後面插入笑出眼淚的表情符號？我不知道要去哪裡點。謝謝。

**Reply by enoch-lightning** [reply]
雖然現在我還不知道去哪裡點，可以插入表情符號。但，我發現[markdown表情圖案](https://www.webfx.com/tools/emoji-cheat-sheet/)中，有一堆表情與對應的markdown，即md語法。

**Reply by EricAgumonJou** [reply]
@enoch-lightning 我自己是用我蘋果電腦內建的表情符號輸入😅

**Comment by Jie2222** [comment]
結合不同領域專長，以低成本方式解決環境問題，做自己有熱情的事，厲害

**Comment by enoch-lightning** [comment]
@EricAgumonJou _您總是能找到新鮮且實用的文章，可以請您分享一下搜尋的方式嗎？是有特別用什麼**關鍵字**，或是用**英文**搜尋嗎？

**Reply by EricAgumonJou** [reply]
我自己是會上IEEE的網站找論文來看, 接著搜尋英文關鍵字看看有沒有比較潛顯且易懂的文章~

**Reply by enoch-lightning** [reply]
哇！好厲害，我下次也來試試看。謝謝你的分享。

**Comment by chupochun** [comment]
原來人工智慧還能如此運用在拯救動植物身上

**Comment by xkllkx** [comment]
藻類對海洋生態平衡真的十分重要，真的對研發團隊感到十分佩服

**Comment by pekoranoinu** [comment]
藻礁真的重要，但是公投沒過，哀。

**Comment by bloodtrail0511** [comment]
我一直都沒想過機器學習可以如此應用在生態保育，或許還有更多可能的應用環境等待我們去發掘

**Comment by awkward-willy** [comment]
海藻對於全球的環境有重大的影響
若能透過科技去保育，想必能有更好的成效
同樣的，這項技術也可以運用在各式各樣的物種上
一定能讓地球上的各生物更祥和的同住

**Comment by Lai27330** [comment]
科技進步的同時感覺也一直在損耗地球資源，能透過機器學習來保護海藻的方式以應對氣候變化，相信未來在面對環境危機時也能有更多解決的轉機。

**Comment by HaoFish123** [comment]
感謝分享，希望地球的環境可以越來越好

**Comment by treeann** [comment]
拯救成功的話真的會有很大的成就感

**Comment by sys511613** [comment]
機器學習的應用真的很廣泛，感覺跟各個看似不相關的領域都漸漸出現連結

**Comment by stan890425** [comment]
機器學習運用在海洋生態，也可兼顧保育

**Comment by jcpan821** [comment]
藻礁真的很重要

**Comment by alexthebigpp** [comment]
希望也可以用來保護其他保育類動物。

**Comment by 110409538** [comment]
希望也可以用來拯救珊瑚.

**Comment by Ben6uo** [comment]
那樣就可以預防紅潮了 就不用怕突然死一堆魚了

**Comment by minniewei** [comment]
我希望相關的技術能越來越被發現

**Comment by bloodtrail0511** [comment]
希望研究團隊能有重大突破!

**Comment by Lynn12378** [comment]
希望科技可以更好的保護環境

**Comment by Jimmy911218** [comment]
謝謝分享真的ai是好是壞要看用的仁和用的事物
